packages/core/src/telemetry/clearcut-logger/event-metadata-key.ts

```diff
@@ -416,4 +416,7 @@ export enum EventMetadataKey {
 
   // Logs an event when the user uses the /model command.
   GEMINI_CLI_MODEL_SLASH_COMMAND = 108,
+
+  // Logs a smart edit tool strategy choice.
+  GEMINI_CLI_SMART_EDIT_STRATEGY = 109,
```

Comment:
Can we move this out of the Model Router keys section? Perhaps to the tool call section.